### DISCUSSION TITLE
Weights & Biases Integration

### DISCUSSION THREAD

**Discussion by ammarsani-my** [azavea/raster-vision#1471] [General]
Hi @AdeelH and @lewfish.

I have run hundreds of experiment with this framework for semantic segmentation. Tq for providing this framework. We found it promising to solve our problem with segmenting on our raster data.

However, my team right now just decided to utilise W&B for centralise our model development. May I know if there is a way I can use it with Raster Vision?

Thanks

#wandb #semanticsegmentation

**Comment by AdeelH** [comment]
RV does not have any built-in integration with W&B. We have not used W&B ourselves, so cannot say how to go about it. We'd be happy to help with any specific questions you run into while going about this.
